<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-UFQCNOCD" ignored="0" name="Test" tests="15" failures="2" timestamp="09 Apr 2019 11:00:23 GMT" time="14.32" errors="0">
  <testcase name="test" time="1.064" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.655" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.658" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.658" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.625" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.653" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.646" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.647" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.619" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.626" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.482" classname="com.qa.tests.APITest">
    <failure type="java.lang.AssertionError" message="expected [201] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [400]
at com.qa.tests.APITest.test(APITest.java:56)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.646" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="0.476" classname="com.qa.tests.APITest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at com.qa.tests.APITest.test(APITest.java:56)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.452" classname="com.qa.tests.APITest"/>
  <testcase name="test" time="3.621" classname="com.qa.tests.APITest"/>
</testsuite> <!-- Test -->
